assets/helpers/tracking/guTracking.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+// ----- Campaigns ----- //
+
+const campaigns : {
+  [string]: string[],
+} = {
+  baseline_test: [
```

Comment:
This seems a little bit confusing.  intCmp is internal campaign code, which implies one campaign per code, but here you have `campaign` as a collection of campaign codes.  If this concept is used elsewhere in the business, that's fine.  Otherwise I'd prefer we used a different name.  Is the collective noun for campaigns a conflict?

Or since this seems to map to tests should we call it `const tests` that maps from test to campaign code?

Reply:
Here I'm seeing it as our 'baseline test' is a type of campaign that we're running, since we're using the internal campaign codes to track it.

I'm reluctant to use the word 'test' because:

a) I feel like it's a bit generic and is starting to get overloaded.
b) I can see the application as being wider than just tests, we are probably going to want to track and modify content based upon campaigns generally, as contributions do.